### ISSUE TITLE
Suggestion: Visual indication for updated information

### ISSUE THREAD

**Issue by tjiagoM** [PhABC/neuroSummerSchools#8]
I think it should exist some kind of visual information saying what is new since (for example) the last commit or the last week. As this is a long list, when we access it we cannot know whether there's something new since the last time we visited the repository. You could have added some school which deadline is somewhere in May, and I might not see it.

Maybe another possibility is a separate section showing the current schools "accepting applications". I guess that who comes here has two objectives: or search what's available for applying, or see what past summer schools happened in the past. In this sense, two different parts in the README.md seem to me more friendly than just a long list.

**Comment by PhABC** [comment]
Great suggestion and I totally agree. The idea was to have a somewhat exhaustive list that one could just update, but new summer schools are always added!

It might be a bit hard to smartly say when a summer school is new, but putting only the summer schools currently accepting application in the table of content might be a good in between solution. It's not like there are that many summer schools and you are most likely interested in the summer schools added that you can still apply to, correct?

Let me know what you think of this!

**Comment by PhABC** [comment]
@titipata what do you think of this idea for the table of content? The way I set it up was a bit redundant with the actual list.

I was thinking of making a python script to append new school directly via command line (updating table at the same time) where it could sort based on deadline as well (should reformat deadlines to dd/mm/yy to make sorting easier). Could be something like ; scan the file, build dictionary, sort, update file.

**Comment by titipata** [comment]
@PhABC, I think that's a great idea to make Python script to do that!

```bash
>> python add_conference.py
conference name: ...
deadline: ...
details: ...
```

I have to think about it a little more on how we should set them up so we don't have a bunch of `.py` files in the repository.

Another way is that we can also [add `.github` file](https://github.com/blog/2111-issue-and-pull-request-templates) in order to give contribution instructions when people want to do Pull Request.  That might be a little easier because people who wants to add details might not prefer to add conference use command line.

I totally agree again that we should have field for deadline/occur date (at least in a structure that we can write code to extract) for each summer schools so that we can sort them by time or by summer school name.

**Comment by PhABC** [comment]
I could write the instruction file, which could be the README file actually. Good idea! If you don't mind the current deadline format than sure we can keep it this way. Let me know if you want some help with the coding as well!

**Comment by tjiagoM** [comment]
@PhABC yes, having a list with the ones currently accepting seems the best way to me. I also think that it is important to have a list with past summer schools.  About the "new" added summer schools I guess I was thinking in something like those old "new" images when a new thing is added, but I'm not sure whether it would have a nice result.

**Comment by titipata** [comment]
@tjiagoM this is actually a great idea. Maybe we can highlight those schools in table of contents. One draw back here is that we have to update them once a month (things like that)

**Comment by Vekkle** [comment]
Hello!
Sorry, I am new to GitHub. How can I suggest a summer school to be added to this list?
Thank you very much for the idea and your realization, by the way.

I want to suggest Summer neurolinguistics school 2017 in Moscow, Russia:
https://www.hse.ru/neuroling/summer_school_2017
Registration fee is appr. 8 € for students and appr. 16 € for non-student participants; university dormitories are available for foreigners for appr. 3 € per night. Organising comittee may help with visa.
Deadlines: 15 May 2017 — poster submission, 1 June 2017 — registration.

**Comment by titipata** [comment]
thanks @vekkle! I think I can add the conference later on today to the repository. Bug me if I didn't do that :)

**Comment by PhABC** [comment]
Added the school @vekkle ! Thank you :)

**Comment by tjiagoM** [comment]
I just thought that even better than a list of summer schools and respective deadline, which we would have to manually update everytime the deadline has passed, it is possible to have a webpage. Github allows each user to have a public github.io webpage, thus we could just have some place in the repository with the schools, and some javascript code would be responsible to show the ones currently active according to the day of the year, and put somewhere else in the webpage the past schools... 
So, the only work we would have would be to manually push information about summer schools to the repository. 
Also, you would have a url which would be the "standard" for this, and more easy to share.

What do you think?
It is very easy to grab a simple html template like these: https://html5up.net/
More info: https://pages.github.com/

**Comment by PhABC** [comment]
This is a great suggestion and it would indeed be quite easy to implement. Could save summerschools in a .json file or something.

**Comment by titipata** [comment]
@PhABC you can set `docs` folder to Github pages in settings page. My suggestion is to use Jekyll page and store each conferences in yml files.
